### PR TITLE
Support multisig wallet

### PR DESCRIPTION
**Pull request by scottphc**
Support multisig wallet, like https://github.com/hippospace/aptos-wallet-adapter/pull/69

Before we integrate with Blocto, we need to change the interface first.

### CODE REVIEW

**Comment by 0xmaayan** [comment]
@scottphc LGTM, thanks!

Could you also add it to the changelog so we can bump and publish a new version.
can follow https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/CONTRIBUTING.md#changesets

**Comment by scottphc** [comment]
@0xmaayan added!

**Comment by scottphc** [comment]
> @scottphc LGTM, thanks!
> 
> Could you also add it to the changelog so we can bump and publish a new version. can follow https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/CONTRIBUTING.md#changesets

@0xmaayan Added!

**Review by 0xmaayan — APPROVED** [review]

File: packages/wallet-adapter-core/src/types.ts [hunk 1]
```diff
@@ -11,7 +11,8 @@ export type NetworkInfo = {
 
 export type AccountInfo = {
   address: string;
-  publicKey: string;
+  publicKey: string | string[];
+  minKeysRequired?: number
```
Comment (kent-white) [inline packages/wallet-adapter-core/src/types.ts:15]: I've had this question for a while but why do we need `minKeysRequired`?

Shouldn't that be coming from on-chain?

If you set minKeysRequired coming back from the wallet an attacker could easily just pass "1"

Comment (scottphc) [reply packages/wallet-adapter-core/src/types.ts:15]: Where can I get minKeysRequired on chain? As I know, there is an Auth Key only. According to https://aptos.dev/concepts/accounts#multisigner-authentication, developers can verify it by using `sha3-256(p_1 | . . . | p_n | K | 0x01)` and then comparing with Auth Key on chain.

**Pull request merged**